lib/spectron.d.ts

```diff
@@ -118,15 +118,19 @@ declare module 'spectron' {
     ): Promise<AccessibilityAuditResult>;
   }
 
-  export interface SpectronWindow extends Electron.BrowserWindow {
-    capturePage(): Promise<Electron.NativeImage>;
-  }
+  export type SpectronWindow = {
+    [P in keyof Electron.BrowserWindow]: Electron.BrowserWindow[P] extends (
+      ...args: infer A
+    ) => infer R
+      ? (...args: A) => Promise<R>
+      : undefined;
+  };
```

Comment:
I fixed this more completely in #999, using `Promise<R extends PromiseLike<infer T> ? T : R>` to avoid doubly-wrapped `Promise`s, and wrapping every module in `Electron`, not just `BrowserWindow`.

Reply:
@andersk Great works!